index.html

```diff
@@ -1735,7 +1735,7 @@ <h2>Web Thing</h2>
                 even within a single Thing. The security configuration
                 aspect of a Thing represents the mechanisms used to
                 control access to the <a>Interaction Affordances</a> and the management of
-                related public and private security configurations.
```

Comment:
Note that technically this expands to "public security data and private security data", whereas elsewhere we used "public security metadata".   We try in general to use metadata when there is a description of something rather than actual content.  This is an exception, but we discussed it and decided it was acceptable.